api-consumption/src/main/java/org/zalando/nakadi/service/subscription/state/CleanupState.java

```diff
@@ -40,7 +40,6 @@ public void onEnter() {
             } catch (final IOException e) {
                 getLog().error("Unexpected fail to release zk connection", e);
             }
-            getContext().getCurrentSpan().finish();
```

Comment:
Seems that it was a wrong place to call `finish()`, but the place in `streamInternal()` is also not a good one (who puts this span there?).